synapse/tests/test_types.py

```diff
@@ -452,29 +457,46 @@ def test_type_str_strip(self):
         self.eq(tlib.getTypeCast('str:lwr', ' ASDF  '), 'asdf')
 
     def test_type_xref(self):
```

Comment:
Can we add another test where the `xref:strval` internally contains an `=`? Is that allowed?

Reply:
that's totally allowed so i've added a test to do that!

Reply:
thanks!